hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestAlterTableDropPartition.scala

```diff
@@ -172,4 +226,51 @@ class TestAlterTableDropPartition extends TestHoodieSqlBase {
       }
     }
   }
+
+  Seq(false, true).foreach { hiveStyle =>
+    test(s"Purge drop multi-level partitioned table's partitions, isHiveStylePartitioning: $hiveStyle") {
```

Comment:
would you please also add a test about drop partition purge for non-partitioned table?